### PR TITLE
migraphx - gdf example mnist

### PR DESCRIPTION
**Pull request by LakshmiKumar23**
Will work once @asalmanp  pushes his changes
Takes an mnist image and calls graphx node on it.
@kiritigowda ready to be merged

### CODE REVIEW

File: tests/amd_migraphx_test/gdf/mnist_migraphx.gdf [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+data b = scalar:FLOAT32,0.0
+data reverse_channel_order = scalar:BOOL,0
+data image_tensor = tensor:4,{28,28,1,1},VX_TYPE_FLOAT32,0
+node com.amd.nn_extension.convert_image_to_tensor input image_tensor a b reverse_channel_order
+
+data model = scalar:STRING,"mnist-8.onnx"
+data output_tensor = tensor:2,{10,1},VX_TYPE_FLOAT32,0
+
+node com.amd.amd_migraphx_node model image_tensor output_tensor
+write output_tensor out.f32
```
Comment (asalmanp) [inline tests/amd_migraphx_test/gdf/mnist_migraphx.gdf:16]: please change the out.f32 to mnist_out.f32. As we will be adding additional gdfs, it's better to have unique names for the outputs.

File: tests/amd_migraphx_test/gdf/out.f32 [hunk 2]
```diff
@@ -0,0 +1 @@
+�pl�H�?|ɓ���5+iA3�%��\i���@��o@�BA
```
Comment (asalmanp) [inline tests/amd_migraphx_test/gdf/out.f32:1]: do we need to keep this file in the repo? is this supposed to be a golden result? if so, for which digit?

**Review by asalmanp — CHANGES_REQUESTED** [review]

File: tests/amd_migraphx_test/gdf/out.f32 (hunk 2 above)
Comment (LakshmiKumar23) [reply tests/amd_migraphx_test/gdf/out.f32:1]: @asalmanp Pavel likes to keep a golden result. I can change the name to reflect it. Do we not want to keep this at all?

Comment (asalmanp) [reply tests/amd_migraphx_test/gdf/out.f32:1]: we can keep it but it needs to have a unique name, and please also mention it in the readme that this is a golden result for a specific digit.

**Review by asalmanp — APPROVED** [review]

**Review by kiritigowda — APPROVED** [review]

**Pull request merged**